### DISCUSSION TITLE
How does the network incentivize participants to join?

### DISCUSSION THREAD

**Discussion by Fatman13** [bluesky-social/atproto#1870] [General Discussion]
As answered [here](https://atproto.com/guides/faq#is-the-at-protocol-a-blockchain)...

> It's not a blockchain nor does it use a blockchain.

I am wondering how the network would incentivize PDS and ~~BGS~~ Relay providers to join and stay in the network in terms of protocol adoption? Or maybe the hardware requirement would be fairly low and network could be bootstraped by a group of enthusiasts? Or all the infra are already being taken care of by an organization?

**Comment by DavidBuchanan314** [comment]
Having thought about the same question, here's my 2 cents. By the way, BGSes have recently been renamed to Relays.

Running a PDS is relatively cheap and easy - very accessible to enthusiasts, volunteers, or other benevolent orgs.

The question for Relays is harder to answer. A Relay that only crawls a subset of PDSes (e.g. tracking only the repos of your local social graph) is doable at hobbyist scale, but a full-network Relay is going to be expensive.

Bluesky PBC is hosting "the" bluesky Relay for now, but hopefully other orgs join them in running more. I'm not sure what the incentives for that are though, beyond sheer benevolence :/

**Comment by bnewbold** [comment]
PDS: intention is for it to be pretty cheap and commodity for most accounts. Self-organized hosting is possible (via social connections, or small payment), or free hosting could be easily subsidized by larger hosting services (freemium, to encourage network growth, etc). There are privacy and "in whose interest" benefits to having a PDS host who has incentives aligned with users.

Relay: the underlying motivation is reliable bulk access to network content. App (AppView) developers and operators in particular will want reliable Relay access. They could run their own, or pay for an SLA (and support, etc) from an existing Relay. The existence of multiple full-network Relays to meet these needs seems a safe bet.

The harder question is "who would give away Relay firehose access to hackers for free". For an already-sustainable Relay, the additional per-subscriber cost should always be a small fraction of resource cost, so a degree of subsidy seems likely (especially if it increases the value of the overall network). The marginal cost of providing the firehose could approach the marginal cost of consuming it (if just pure bandwidth). Tons of no-cost subscribers could end up being a substantial cost some day. One model could be cloud platforms giving no-cost subsidized access to consumers, similar to DNS or AWS S3 "public datasets". Or a mirror network of public-interest fan-out nodes that re-broadcast a firehose, similar to linux package distro mirror networks. Most likely some combination of models.

**Reply by FLPresets** [reply]
Is there an incentive for users to join/run a PDS outside privacy?  

Maybe I misunderstand- is there a community benefit to joining/running a PDS, like access to a server feed/chat?  Or do you think the privacy niche will be big enough?

**Reply by DavidBuchanan314** [reply]
My personal incentive for running a PDS is data and cryptographic "self-sovereignty". i.e. I'm the one responsible for hosting my own data, and more importantly, I hold my own repo signing keys. It provides strong guarantees that nobody else can delete or modify my data, or make posts pretending to be me. Unfortunately this is also a fairly niche interest in the grand scheme of things, but I certainly won't be the only one - I expect it'll be a similar demographic to those who host their own blogs.

**Reply by FLPresets** [reply]
I run a blog, and think the demographic will be much narrower than that. The most successful blogs leverage their platform to build a discoverable community, then monetize.  There's no mechanism for that here.

To my knowledge, every decentralized social network that succeeds - from web forums to decentralized governments to terrorist networks- they all cater to autonomous niche communities.  Fostering autonomous niche communities is the hallmark of decentralized success. 

But it looks like AT Protocol doesn't intend to cater to autonomous niche communities.  Interesting.

Thanks for the clarity though!  Very informative about the direction AT is taking.

**Reply by devinivy** [reply]
I think it's worth mentioning that the signing key—which we think of as being owned by the pds—plays a bit of double-duty: it signs your repository, but it also helps prove to other services that you're you.  So for example, requests made to the appview or a feed generator know the request is coming from you because you've signed some credentials with your private signing key.  So the pds is really the locus of your identity on the network, and I think this will make self-sovereignty even more desirable for some folks.

**Reply by devinivy** [reply]
Ah we posted at the same time!  I will add that in my personal view the core point of the pds is that it allows us to unbundle hosting of the network's social data from any one application/platform, which is what makes the data on the network open.  I hope that niche communities will be able to make good use of this open social ecosystem, whether or not it involves them hosting their own pds.  I also wouldn't be surprised if some use cases cause us to look at the pds in new ways—there are some interesting designs for forums, for example, that could leverage the pds a little bit differently than we do for Bluesky's global/broadcast microblogging.

**Reply by pfrazee** [reply]
One other good reason to run a PDS might be if you're running custom code around it. You might, for instance, wrap an API around an account to add behaviors (eg for bots or future subprotocols). You might also be running a website around the account and want to have local queries of your content to render on the pages.
